src/lib/format/l20n/ast/parser.js

```diff
@@ -7,18 +7,20 @@ const MAX_PLACEABLES = 100;
 
 export default {
   parse: function(env, string, pos = false) {
```

Comment:
Can we choose an unambiguous and future-proof name and make it the only exported symbol right now?